login_with/google/oauth_10_google_api.e

```diff
@@ -1,12 +1,14 @@
 note
 	description: "Summary description for {OAUTH_10_GOOGLE_API}."
-	author: ""
 	date: "$Date$"
 	revision: "$Revision$"
+	EIS: "name=Google API OAuth 1.0", "src=https://developers.google.com/identity/protocols/OAuth_ref", "protocol=uri"
 
 class
 	OAUTH_10_GOOGLE_API
 
+obsolete " OAuth 1.0 was officially deprecated on April 20, 2012, and is no longer supported. We encourage you to migrate to OAuth 2.0 {OAUTH_20_GOOGLE_API} as soon as possible."
```

Comment:
Eiffelstudio should have reported a warning, since this obsolete message does not have any date in expected format.